File: AppGenerateFiles/dbo/Model/Transaction_Egresos.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Transaction_Egresos : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? numero_egreso { get; set; }
       public Double? monto { get; set; }
       public DateTime? fecha { get; set; }
       public string? descripcion { get; set; }
       public string? nombre { get; set; }
       public string? banco { get; set; }
       public string? anulado { get; set; }
       public string? observaciones { get; set; }
       public Double? tc { get; set; }
       public Double? dolar { get; set; }
       public DateTime? fanulado { get; set; }
   }
}

File: AppGenerateFiles/dbo/FrontModel/Transaction_Egresos.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools, BasicStates } from "../../WDevCore/WModules/WComponentsTools.js";
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
class Transaction_Egresos extends EntityClass {
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ numero_egreso;
   /**@type {Number}*/ monto;
   /**@type {Date}*/ fecha;
   /**@type {String}*/ descripcion;
   /**@type {String}*/ nombre;
   /**@type {String}*/ banco;
   /**@type {String}*/ anulado;
   /**@type {String}*/ observaciones;
   /**@type {Number}*/ tc;
   /**@type {Number}*/ dolar;
   /**@type {Date}*/ fanulado;
}
export { Transaction_Egresos }
